api/app/controllers/spree/api/base_controller.rb

```diff
@@ -42,8 +44,24 @@ def permitted_line_item_attributes
         end
       end
 
-      def content_type
-        case params[:format]
+      protected
+      # So it will be available in serializer
+      helper_method :current_api_user
+
+      private
+
+      def pagination(collection)
+        {
+         count: collection.count,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.